Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2404,9 +2458,12 @@ extension BrowserViewController: KeyboardHelperDelegate {
 
     func keyboardHelper(keyboardHelper: KeyboardHelper, keyboardWillHideWithState state: KeyboardState) {
         keyboardState = nil
-        // if we are showing snack bars, adjust them so they are no longer sitting above the keyboard
-        if snackBars.subviews.count > 0 {
-            adjustFooterSize(nil)
+        updateViewConstraints()
+
+        UIView.animateWithDuration(state.animationDuration) {
+            UIView.setAnimationCurve(state.animationCurve)
+            self.findInPageContainer.layoutIfNeeded()
+            self.snackBars.layoutIfNeeded()
```

Comment:
I think we always need to use `layoutIfNeeded` inside of `animateWithDuration` for constraints to be animated, right?